### ISSUE TITLE
Improve the check for completion

### ISSUE THREAD

**Issue by rvhonorato** [haddocking/benchmark-tools#16]
With the current check;

```python
                if Path(self.path, "structures/it1/water/file.list").exists():
                    current_status = "complete"
```

We are unable to correctly assigned the `done` state for scenarios in which no itw is executed.

**Comment by rvhonorato** [comment]
no longer relevant after #20
